docs/media-app.md

```diff
@@ -0,0 +1,39 @@
+# The Media app
+
+## User-visible features
+
+The media app provides file and image management to the Django admin.
+It also integrates with the CMS's WYSIWYG text editor to provide a file browser and image browser interface that allows images and files to be added directly into the editor.
+
+The default `FileAdmin` adds a thumbnail preview to the list view, falling back to an appropriate icon for the file type if the file is not an image.
+
+For images, there is an in-browser image editor that gives quick access to common image operations such as cropping and rotating.
+
+## Models
+
+### File
+
+`cms.apps.media.models.File` is a wrapper around a Django FileField. This allows users to upload their files in one place and use it in more than one place.
+
+The CMS's `File` provides additional fields: a title, alt text (for images), attribution and copyright. It is up to you how, or if, to render these on the front-end of the website.
+
+`File` is not typically used for files uploaded via the public front-end of a website (i.e. non-staff users). For this, you'll want to use a simpler Django `FileField` or `ImageField`.
+
+### Label
+
+`cms.apps.media.models.Label` helps administrators organise media; think of them as tags, or notes to self. They are not intended to be shown to users on the front end of a website.
+
+### Video
+
+`cms.apps.media.models.Video` is a collection of video files and related imagery.  You can use it to easily create cross-browser compatible ``<video>`` tags on the frontend of your website.
```

Comment:
I'm unsure of how deep these docs are going to go. Do you want to try and document helper functions that models have or are these docs more of an overview of the features that the CMS has?

Reply:
That is undecided at this point!